terralab/logic/pipeline_runs_logic.py

```diff
@@ -75,6 +76,41 @@ def get_pipeline_run_status(
         return pipeline_runs_client.get_pipeline_run_result(pipeline_name, str(job_id))
 
 
+def get_pipeline_runs(n_results_requested: int) -> list[PipelineRun]:
+    """Get all the pipeline runs a user has submitted"""
+
+    with ClientWrapper() as api_client:
+
+        pipeline_runs_client = PipelineRunsApi(api_client=api_client)
+
+        n_results_chunk = 10  # in case they request a ton
```

Comment:
10 is the api default. is it worth choosing the smaller of that and the number requested, even when we'll still want to trim the list at the end? (which we need to do regardless in case they request say 15 jobs and have more than that)